lib/cdo/delete_accounts_helper.rb

```diff
@@ -97,12 +103,24 @@ def self.purge_orphaned_students(user_id)
     end
   end
 
-  # Remove all user generated content associated with any PD the user has been through.
+  # Remove all user generated content associated with any PD the user has been through, as well as
+  # all PII associated with any PD records.
   # @param [Integer] The ID of the user to clean the PD content.
   def self.clean_and_destroy_pd_content(user_id)
     PeerReview.where(reviewer_id: user_id).each(&:clear_data)
 
     Pd::TeacherApplication.where(user_id: user_id).each(&:destroy)
+    Pd::FacilitatorProgramRegistration.where(user_id: user_id).each(&:clear_form_data)
+    Pd::RegionalPartnerProgramRegistration.where(user_id: user_id).each(&:clear_form_data)
+    Pd::WorkshopMaterialOrder.where(user_id: user_id).each(&:clear_form_data)
+
+    # TODO(hamms): This query is not indexed. Add an index so that it is.
+    pd_enrollment_id = Pd::Enrollment.where(user_id: user_id).pluck(:id).first
+    if pd_enrollment_id
+      Pd::TeacherconSurvey.where(pd_enrollment_id: pd_enrollment_id).each(&:clear_form_data)
+      Pd::WorkshopSurvey.where(pd_enrollment_id: pd_enrollment_id).each(&:clear_form_data)
+      Pd::Enrollment.where(id: pd_enrollment_id).each(&:clear_form_data)
```

Comment:
And here: [clear_data](https://github.com/code-dot-org/code-dot-org/blob/b1ef632a91952668e67ef62badf80a97505b84aa/dashboard/app/models/pd/enrollment.rb#L246)

Reply:
Done. Sorry for sloppiness.